main/res/values/strings.xml

```diff
@@ -826,6 +826,8 @@
     <string name="init_summary_useInternalRouting">Use c:geo internal routing to be able to use automatic routing data downloading and other functions</string>
     <string name="init_bigSmileysOnMap">Big icons</string>
     <string name="init_summary_bigSmileysOnMap">If enabled, symbols like the \'found smiley\' or the \'corrected coordinates\' marker will be shown enlarged on the map instead of the cache type icon (requires restart).</string>
+    <string name="init_dtMarkerOnCacheIcon">D/T rating on cache icons</string>
+    <string name="init_summary_dtMarkerOnCacheIcon">Show an marker for the Difficulty and Terrain ratings on cache markers in upper-right corner ("stored" marker is moved to middle) (requires restart).</string>
```

Comment:
Can we add a small sentence to the summary which half of the circle belongs to which of both values?

Reply:
done